gridstatus/nyiso.py

```diff
@@ -870,7 +872,13 @@ def _download_nyiso_archive(
                     log(msg, verbose)
                     continue
                 df = pd.read_csv(z.open(csv_filename))
-                df["File Date"] = d.normalize()
+
+                # Get the modified date of the file
+
+                df["File Date"] = datetime(
```

Comment:
why not use pandas here like we do elsewhere?

Reply:
```python
pd.read_html("http://mis.nyiso.com/public/P-7list.htm", skiprows=2)
```

<img width="630" alt="image" src="https://github.com/kmax12/gridstatus/assets/371792/27b34bc7-96de-4305-8fd6-2b14d63979b1">

Reply:
Yeah, I haven't got the non-zip file publish time yet.

Reply:
Thanks, updated to use that file list.